schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -50,8 +50,25 @@ properties:
     properties:
       mailAddress:
         type: string
+      timeout:
+        type: integer
       credentials:
         "$ref": "/common-1.json#/definitions/vaultSecret"
+    required:
+    - mailAddress
+    - timeout
```

Comment:
any particular reason for making this a requirement or should the code provide the current value as in case there's nothing here?

Reply:
I've made these required because they are also required in the graphql schema. So far, the code doesn't have any defaults, and I think it is better to set it explicitly.

Reply:
Just talking about `timeout`. I agree `mailAddress` being mandatory

Reply:
I see; there are pros and cons as usual. I will make it non-required and set 30 seconds as the default